### PR TITLE
feat: Adding telemetry using influxdb and telegraf

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [ x] Added label to the Pull Request for easier discoverability and search
- [ x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

1. make build-telegraf
2. make INFLUXPASS=password run-telegraf
3. Click on "Build a Dashboard", then "Import dashboard" and select the file intel_core_and_igpu_telemetry.json under telegraf folder.
4. [127.0.0.1:8006](http://127.0.0.1:8086/)

### CODE REVIEW

File: telegraf/docker-run-influxdb.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	-e DOCKER_INFLUXDB_INIT_USERNAME=telegraf \
+	-e DOCKER_INFLUXDB_INIT_ORG=telegraf \
+	-e DOCKER_INFLUXDB_INIT_MODE=setup \
+	-e DOCKER_INFLUXDB_INIT_BUCKET=telegraf-bucket \
+	-e INFLUXD_SESSION_RENEW_DISABLED=true  \
+	-e INFLUXD_SESSION_LENGTH=700 \
+	--name influxdb2 \
+	-itd --rm influxdb:2.6.1
+
+# Bug: unauth: -v `pwd`/influxdb2-cache:/var/lib/influxdb2 \
```
Comment (jim-wang-intel) [inline telegraf/docker-run-influxdb.sh:20]: remove this dead code

File: telegraf/docker-run.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+./docker-run-influxdb.sh
+sleep 5
+#docker exec influxdb2 influx auth create --all-access --org telegraf
+sleep 5
```
Comment (jim-wang-intel) [inline telegraf/docker-run.sh:11]: remove

**Review by jim-wang-intel — COMMENTED** [review]
some clean up and also please resolve the review dog issues

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by brian-intel — APPROVED** [review]

File: docs_src/telemetry/setup.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Import Template
+
+Click on "Build a Dashboard", then "Import dashboard" and select the file **intel_core_and_igpu_telemetry.json** under telegraf folder.
+
+![Import template](../images/import.png)
+
+## Run Dashboard
+
+![dashboard](../images/dashboard.png)
```
Comment (NeethuES-intel) [inline docs_src/telemetry/setup.md:31]: you could add the clean commands here

**Review by NeethuES-intel — COMMENTED** [review]
UI worked. I didn't see any telemetry data on the dashboard. Should I set up anything else ?

Also, if influxdb is already running, then error is not shown to user when running the make run command.

**Review by NeethuES-intel — APPROVED** [review]
Dashboard worked after changing influxdb password

**Pull request merged**